apps/web/pages/apps/installed/[category].tsx

```diff
@@ -179,7 +179,9 @@ const IntegrationsContainer = ({ variant, exclude }: IntegrationsContainerProps)
                 })}
                 description={t(`no_category_apps_description_${variant || "other"}`)}
                 buttonRaw={
-                  <Button color="secondary" href={variant ? `/apps/categories/${variant}` : "/apps"}>
+                  <Button
+                    color="secondary"
+                    href={variant ? `/apps/categories/${variant}` : "/apps/categories/other"}>
```

Comment:
Added the `/categories/other` to the false part of this shorthand if statement as that's the correct place when clicking the button in the "Other" empty state screen.